drivers/hmis/app/models/hmis/hud/custom_client_address.rb

```diff
@@ -38,7 +38,6 @@ class Hmis::Hud::CustomClientAddress < Hmis::Hud::Base
 
   def equal_for_merge?(other)
     columns = [
-      :address_type,
```

Comment:
This changes logic for client merge. We'll no longer keep 2 addresses if they have the same exact values but differing "type" (type can be postal/physical/both)